startup-script.sh

```diff
@@ -9,7 +9,12 @@ compile() {
 
 run_stats() {
   echo -e '\nRunning stats...\n'
-  rm -rf ~/.quai
+  if [[ "$OSTYPE" == "darwin"* ]];
```

Comment:
since you are bringing in a feature you can squash this commit with the first one.